src/main/java/app/controllers/UserController.java

```diff
@@ -54,7 +54,7 @@ public static void login(Context ctx, ConnectionPool connectionPool) {
             ctx.sessionAttribute("currentUser", user);
             // Hvis ja, send videre til forsiden med login besked
             ctx.attribute("message", "Du er nu logget ind");
-            ctx.render("index.html");
+            ctx.render("user-frontpage.html");
         } catch (DatabaseException e) {
             // Hvis nej, send tilbage til login side med fejl besked
             ctx.attribute("message", e.getMessage());
```

Comment:
Maybe have an error message attribute so not to be confused with normal messages like "Your account was successfully created"